templates/scenarios/sso-tab-js/.fx/configs/config.dev.json.tpl

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Just curious, is the tpl file a new design in V3? I don't seem to see it in the current template

Reply:
No. We add .tpl suffix to identify which file includes placeholders. You can find such files in v2 templates, too.
https://github.com/OfficeDev/TeamsFx/blob/dev/templates/tab/csharp/default/Interop/InteropModuleBase.cs.tpl

Reply:
OK, I got it. It looks like most of it is used in csharp's template